.github/renovate.json

```diff
@@ -7,22 +7,47 @@
   "schedule": ["every weekday before 5am"],
   "packageRules": [
     {
-      "enabled": false,
       "matchManagers": ["npm"],
       "rangeStrategy": "bump",
       "stabilityDays": 3
     },
+    {
+      "enabled": false,
+      "matchDepTypes": ["dependencies", "peerDependencies"],
+      "matchManagers": ["npm"]
+    },
+    {
+      "enabled": true,
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": [
+        "^axios$",
+        "^chalk$",
+        "^core-js$",
+        "^envalid$",
+        "^execa$",
+        "^lit$",
+        "^webpack-cli$",
+        "^webpack-dev-server$"
+      ]
+    },
```

Comment:
These deps appear in `devDependencies` for some workspace but are `dependencies` in others. I have updated them in https://github.com/blockprotocol/blockprotocol/pull/703. This rule makes Renovate covers _some_ prod dependencies and avoids failures in `yarn lint:dependency-version-consistency`. Let's see how it goes.